README.md

```diff
@@ -0,0 +1,90 @@
+# Inside
+
+Inside is an open source platform to deal with all the intern process of a company related to its employees. Our goal is to make easy to keep a track and manage all information related to the people that work **inside** the company.
+
+| Master | Development |
```

Comment:
Please, follow this layout:
https://github.com/ifactory-solutions/inside-client/tree/develop

Reply:
Actually, I just copy and pasted this README... :|

Reply:
Are you sure you have the latest version locally? 🤔

Reply:
Copied the raw code directly from the repository at GitHub.

I just changed the techs used, renamed "inside-client" to "inside-server" and added the Codacy badges. What is different from there?

Reply:
Ok, I just saw that it was updated after I started working on it. Going to make some changes.